user_docs/en/changes.t2t

```diff
@@ -42,7 +36,7 @@ What's New in NVDA
   - Allows splitting NVDA sounds in one channel (e.g. left) while sounds from all other applications in the other channel (e.g. right).
   - Toggled by ``NVDA+alt+s``.
   -
--
+- - Added support for the BrailleEdgeS3 braille device(#16279, @EdKweon).

```

Comment:
I would suggest amending the line "Added support for the BrailleEdgeS2 braille device." to "Added support for the BrailleEdgeS2 and BraillEdgeS3 braille devices." and adding both PR numbers

Reply:
Once that's done this can be merged

Reply:
I ammended PR. Thanks!